plugins/processors/cluster_aggregator.go

```diff
@@ -35,6 +37,10 @@ func (this *ClusterAggregator) Process(batch *metrics.DataBatch) (*metrics.DataB
 			}
 		}
 	}
+
```

Comment:
Can revert this file and just add a nil check on the call to process as commented above.